main.py

```diff
@@ -155,6 +217,28 @@ def random_email(name=None):
            random.choices(EMAIL_DATA, emailChoices)[0][1]
 
 
+def audioToText(mp3Path):
+    # convert wav to mp3
+    sound = AudioSegment.from_mp3(mp3Path)
+    sound.export(CAPTCHA_WAV_FILENAME, format="wav")
+
+    with sr.AudioFile(CAPTCHA_WAV_FILENAME) as source:
+        audio_text = speech_recognition_recognizer.listen(source)
+        try:
+            text = speech_recognition_recognizer.recognize_google(audio_text)
+            print('Converting audio transcripts into text ...')
+            return text
+        except Exception as e:
+            print(e)
+            print('Sorry.. run again...')
+
+
+def saveFile(content, filename):
+    with open(filename, "wb") as handle:
+        for data in content.iter_content():
+            handle.write(data)
+
+
 def main():
     while True:
```

Comment:
If desired, the multiprocessing implementation from https://github.com/SeanDaBlack/KelloggBot/pull/21 could be placed here.